frontend/MODULE.bazel

```diff
@@ -34,3 +36,22 @@ rules_ts_ext = use_extension(
 )
 rules_ts_ext.deps()
 use_repo(rules_ts_ext, "npm_typescript")
+
+oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
+
+oci.pull(
+    name = "debian_node",
+    image = "node:20-slim",
+    platforms = [
+        "linux/amd64",
+        "linux/arm64/v8",
+    ],
+)
+use_repo(oci, "debian_node")
+
+node = use_extension(
+    "@rules_nodejs//nodejs:extensions.bzl",
+    "node",
+    dev_dependency = True,
+)
+use_repo(node, "nodejs_toolchains")
```

Comment:
Needed on MacOS under bzlmod to run
```
--extra_toolchains=@nodejs_toolchains//:linux_arm64_toolchain_target
```

WORKSPACE does not have this issue on transitions